src/core/services/TransactionService.ts

```diff
@@ -101,27 +101,36 @@ export class TransactionServiceImpl implements TransactionService {
   public handleTransaction = async ({
     network,
     tx,
+    useExternalService = false,
```

Comment:
I don't think this is needed, it's already undefined by default
```suggestion
    useExternalService,
```

Reply:
much nit?

changed here https://github.com/yearn/yearn-finance-v3/pull/674/commits/cc4c6b9e0a0ee50a21a9454875983308ccc16afc XD

Reply:
It wasn't a nit :) I think it adds a bit of confusing, implying that `useExternalService` is required when it's not